examples/meteor/client/styles/base.css

```diff
@@ -285,9 +284,9 @@ label[for='toggle-all'] {
 	display: block;
 }
 
-#todo-list li .edit {
-	display: none;
-}
+/*#todo-list li .edit {*/
+	/*display: none;*/
+/*}*/
```

Comment:
In the future todomvc-common won't be the only one we need so that kinda sad but I guess this is the way to go for the time being.